src/config/config.py

```diff
@@ -0,0 +1,58 @@
+"""Module to handle configuration application configuration parameters."""
+import configparser
+import logging
+from os.path import expanduser, isfile
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+config_file = "config.ini"
+
+
+def _find_config_directory() -> str:
+    # TODO: Check if windows or linux, use enviroment variables
+    return expanduser("~") + "/.config/nina/"
+
+
+def _get_default_config() -> configparser.ConfigParser:
```

Comment:
Would be nice to add batch size here too.

Reply:
Ah yes, that should be easy to add. Does it make most sense to have that parameter under core?

Reply:
This is now implemented with commit 6b1de6a3dc954d8e5081942d1481e0aea2332f06

However, it can potentially break `VideoLoader` logic regarding progress and how it saves progress. If the `batch_size` gets changed when the application is shut down in the middle of processing a job, it will break. It could potentially re-examine a lot of data, or throw errors for indexes out of range in videos. 

Possible solution to this is to store `batch_size` parameter in `Job` class. `VideoLoader` can then fetch this value from job if it is set. Otherwise it would use the default `batch_size` from the config file, and store the same value in `Job`.

Reply:
crap...

You only need to know the old batch size to figure out which batch to start from, after that it should continue using the new batch size.